### ISSUE TITLE
[BUG]: getCurrentPosition is undefined

### ISSUE THREAD

**Issue by paulrinaldi** [TheWidlarzGroup/react-native-video#3876]
### Version

6.1.1

### What platforms are you having the problem on?

Android

### System Version

API Level 34

### On what device are you experiencing the issue?

Simulator

### Architecture

Old architecture

### What happened?

Try to use the ref to the player .current .getCurrentPosition() as marked in the docs https://thewidlarzgroup.github.io/react-native-video/component/methods#getcurrentposition

```Uncaught Error
videoRef.current.getCurrentPosition is not a function (it is undefined)

onPlaybackStateChanged={(data) => {
console. log( 'data.isPlaying', data.isPlaying); console. log( 'videoRef.getCurrentPosition', vic <----
```

### Reproduction

```
import { StatusBar } from 'expo-status-bar';
import { StyleSheet, Text, View } from 'react-native';
import Video from 'react-native-video';
import { useRef } from 'react';

export default function App() {
  const videoRef = useRef(null);

  return (
    <View style={styles.container}>
      <Video
        ref={videoRef}
        source={{ uri: 'https://www.w3schools.com/html/mov_bbb.mp4' }}
        style={styles.backgroundVideo}
        debug={{
          enable: true,
          thread: true,
        }}
        controls={true}
        onAudioFocusChanged={(hasAudioFocus) => {
          console.log('hasAudioFocus', hasAudioFocus);
          console.log('videoRef.getCurrentPosition', videoRef.current.getCurrentPosition());
        }}
        onPlaybackStateChanged={(data) => {
          console.log('data.isPlaying', data.isPlaying);
          console.log('videoRef.getCurrentPosition', videoRef.current.getCurrentPosition());
        }}
        
      />

    </View>
  );
}
```

**Comment by freeboub** [comment]
I think it will be available in 6.2 ! Sorry, the doc is from master branch, not from the last tag ...

**Comment by freeboub** [comment]
Ok, let's close this issue, I confirm it will be in next version.
There is surely a process issue here ...
